frontend/src/components/teacher/student-management/view-students/EditStudentPopover.tsx

```diff
@@ -1,18 +1,54 @@
 import React from "react";
+import { useMutation } from "@apollo/client";
 import { Divider, useDisclosure, VStack } from "@chakra-ui/react";
 
+import { DELETE_STUDENT } from "../../../../APIClients/mutations/ClassMutations";
+import { GET_CLASS_STUDENTS_BY_ID } from "../../../../APIClients/queries/ClassQueries";
+import type { StudentResponse } from "../../../../APIClients/types/ClassClientTypes";
+import { getQueryName } from "../../../../utils/GeneralUtils";
+import Modal from "../../../common/modal/Modal";
 import Popover from "../../../common/popover/Popover";
 import PopoverButton from "../../../common/popover/PopoverButton";
 
-const EditStudentPopover = (): React.ReactElement => {
+type EditStudentPopoverProps = {
+  student: StudentResponse;
+  classId: string;
+};
+
+const EditStudentPopover = ({
+  student,
+  classId,
+}: EditStudentPopoverProps): React.ReactElement => {
+  const [deleteStudent] = useMutation(DELETE_STUDENT, {
```

Comment:
can we add success / error toast?

Reply:
I think I'll make this change in a follow-up pr. I found a few others that are missing toasts, and a few more that don't use standard modal components. Planning to refactor everything to use the same component and make an easier way of having these toasts

Reply:
sure - can you make a ticket for it?

Reply:
https://www.notion.so/uwblueprintexecs/Add-loading-state-toasts-to-all-modals-7d8198e55ee545ebbe1f096740937b67